### PR TITLE
Fixtures packaging

### PR DESCRIPTION
**Pull request by oleorhagen**
Same, Have a look at the code as a whole, this will be one commit in the end.

### CODE REVIEW

File: tests/acceptance/conftest.py [hunk 1]
```diff
@@ -13,202 +13,16 @@
...
+import os
+import sys
 
-@pytest.fixture(autouse=True)
-def commercial_test(request, bitbake_variables):
-    mark = request.node.get_closest_marker("commercial")
-    if mark is not None and not request.config.getoption("--commercial-tests"):
-        pytest.skip("Tests of commercial features are disabled.")
+cwd = os.getcwd()
+sys.path.append(cwd + "/image-tests/tests")
```
Comment (lluiscampos) [inline tests/acceptance/conftest.py:23]: Same thing as I [commented in mender-convert](https://github.com/mendersoftware/mender-convert/pull/262#discussion_r531133156). I think my suggestion is a bit better as it is independent on the cwd, but it is up to you. Most importantly is to be consistent :)

File: tests/acceptance/test_bootimg.py [hunk 2]
```diff
@@ -17,16 +17,13 @@
 
 import pytest
 
-from common import (
-    build_image,
-    latest_build_artifact,
-)
+from utils.common import build_image, latest_build_artifact
```
Comment (lluiscampos) [inline tests/acceptance/test_bootimg.py:20]: Mmm. Are you using a different `black` version? I am pretty sure I formatted this previously with the tool

**Review by lluiscampos — APPROVED** [review]
This one looks ready!

File: tests/acceptance/conftest.py (hunk 1 above)
Comment (oleorhagen) [reply tests/acceptance/conftest.py:23]: Agreed! adopting

File: tests/acceptance/test_bootimg.py (hunk 2 above)
Comment (oleorhagen) [reply tests/acceptance/test_bootimg.py:20]: I'll check.

**Comment by kacf** [comment]
:tada: This will help me with dunfell as well!

**Pull request merged**